ci/bin/format.dart

```diff
@@ -892,7 +891,15 @@ class DartFormatChecker extends FormatChecker {
       });
     } else {
       final List<WorkerJob> completedJobs = await dartFmt.runToCompletion(jobs);
-      incorrect = completedJobs.where((WorkerJob job) => job.result.exitCode == 1);
+      final List<WorkerJob> incorrectJobs = incorrect = [];
+      for (final WorkerJob job in completedJobs) {
+        if ((job.result.exitCode == 1 && job.result.stderr.isNotEmpty) || job.result.exitCode > 1)  {
```

Comment:
Can't the exit code also be negative?

Seems like you want `job.result.exitCode != 0 && (job.result.stderr.isNotEmpty || job.result.exitCode != 1)`

Reply:
Fixed. Thank you!